main.js

```diff
@@ -27,6 +37,13 @@ function createElementAttribute(name) {
   };
 }
 
+function createStringElementAttribute(name) {
```

Comment:
Do you want to support default values for strings and elements? I.e. check this.hasAttribute(name) and if not then return the default value in the getter?